examples/basic-server-react/src/mcp-app.module.css

```diff
@@ -3,7 +3,9 @@
   --color-primary-hover: #1d4ed8;
   --color-notice-bg: #eff6ff;
 
-  min-width: 425px;
+  width: 100%;
+  max-width: 425px;
```

Comment:
Hmm, I don't think we want `max-width` here, but it looks like this will be addressed by https://github.com/modelcontextprotocol/ext-apps/pull/153.